### DISCUSSION TITLE
Database Auto close

### DISCUSSION THREAD

**Discussion by savidudu** [keepassxreboot/keepassxc#8430] [Q&A] [answered]
Hello Team,

When we are installing the KeepassXC then the autoclose time by default is 240 seconds, We would like to disable or set it to one specific time lets say 5 minutes (300 Seconds).

Basically, We don't want to users to change the auto close timing, Can we do something here so it should not get changed ?

Thanks,

**Comment by droidmonkey** [comment, marked as answer]
This is not currently possible to restrict it from being changed.
